net/lighttpd/Makefile

```diff
@@ -52,41 +52,64 @@ config LIGHTTPD_SSL
 	default y
 	help
 	  Implements SSL support in lighttpd (using libopenssl). This
-	  option is required if you enable the SSL engine in your
-	  lighttpd confguration file.
+	  option is enabled by default for backwards compatibility.
+          Select one TLS module below if you enable the SSL engine in
+	  your lighttpd configuration file.
+          (mod_gnutls, mod_mbedtls, mod_nss, mod_openssl, mod_wolfssl)
 endef
 
+ifneq ($(strip $(CONFIG_LIGHTTPD_SSL)),)
+  ifeq ($(CONFIG_PACKAGE_lighttpd-mod-openssl),)
+  ifeq ($(CONFIG_PACKAGE_lighttpd-mod-mbedtls),)
+  ifeq ($(CONFIG_PACKAGE_lighttpd-mod-wolfssl),)
+  ifeq ($(CONFIG_PACKAGE_lighttpd-mod-gnutls),)
+  ifeq ($(CONFIG_PACKAGE_lighttpd-mod-nss),)
+    CONFIG_PACKAGE_lighttpd-mod-openssl=m
+  endif
+  endif
+  endif
+  endif
+  endif
+endif
+
+ifneq ($(CONFIG_PACKAGE_lighttpd-mod-auth),)
+  ifeq ($(CONFIG_PACKAGE_lighttpd-mod-authn_file),)
+    CONFIG_PACKAGE_lighttpd-mod-authn_file=m
+  endif
+endif
+
+# (lighttpd supports -Dwith_brotli if openwrt provided libbrotli as a package)
```

Comment:
Just add -Dwith_brotli=false below.

Reply:
ok.

(I had added a comment to document the choice.  If brotli were available as a standard package in OpenWrt, then I would add an option for lighttpd to use it)